kuma/validate.go

```diff
@@ -0,0 +1,45 @@
+package kuma
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/layer5io/gokit/smi"
+)
+
+func (h *handler) smiTest(id string) error {
+
+	e := &Event{
+		Operationid: id,
+		Summary:     "Deploying",
+		Details:     "None",
+	}
+
+	annotations := map[string]string{
+		"kuma.io/gateway": "enabled",
+	}
+
+	test, err := smi.New(context.TODO(), "kuma", h.kubeClient)
```

Comment:
could "kuma" be taken from mesh config?

Reply:
Yes @mgfeller  thanks for noticing this. Ill make this change too.